src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const ul = document.querySelector('#thumbs');
+const largeImg = document.querySelector('#largeImg');
+
+ul.addEventListener('click', (e) => {
+  const thumb = e.target.closest('a');
+
+  e.preventDefault();
+
+  getThumb(thumb.href, thumb.tilte);
+});
+
+function getThumb(href) {
+  largeImg.src = href;
+}
```

Comment:
I think this will do

```suggestion
  largeImg.src = thumb.href;
});
```